src/library/scala/collection/immutable/BitSet.scala

```diff
@@ -137,7 +139,7 @@ object BitSet extends BitSetFactory[BitSet] {
         if (elems1 == 0L) throw new NoSuchElementException("Empty BitSet")
         new BitSet2(elems0, elems1 - java.lang.Long.lowestOneBit(elems1))
       }
-      else new BitSet2(elems0 - java.lang.Long.lowestOneBit(elems0), elems1)
+      else createSmall(elems0 - java.lang.Long.lowestOneBit(elems0), elems1)
```

Comment:
here on the other hand, `elems1` should always be non-zero, so we could keep `new BitSet2`